INSTALL.md

```diff
@@ -93,6 +93,44 @@ $ make -f policy_utilities.mak clean
 $ make -f policy_utilities.mak
 ```
 
+If you wish to use the Policy Generator instead of manually composing the
+policies, you need to build it in utilities. The Policy Generator requires the
+json-schema-validator library which in turn depends on the JSON for Modern C++
+library. Follow the following instructions to have them built and installed on
+your system:
+
+```shell
+$ cd $(CERTIFIER)/utilities
+$ git clone https://github.com/nlohmann/json.git
+$ cd json
+$ mkdir build
+$ cd build
+$ cmake ..
+$ make
+$ sudo make install
+$ cd ..
```

Comment:
Maybe leave a blank line before and after this line. And similarly for the `cd ..` on L119 below.

That way it will be clear that the prev set of commands are related to one repo. Blank line will visually separate out the work done for one repo from that done for the 2nd repo.

Reply:
Done